lnd_test.go

```diff
@@ -2744,7 +2744,7 @@ func testGraphTopologyNotifications(net *networkHarness, t *harnessTest) {
 
 	// We should receive an update advertising the newly connected node,
 	// Bob's new node announcement, and the channel between Bob and Carol.
-	for i := 0; i < 3; i++ {
+	for i := 0; i < 4; i++ {
```

Comment:
I noticed that we are receiving two channel announcements in the previous test. This change gets my test working, but I am not sure if it is hiding something that should be a failure.

Reply:
What channels are you getting announcements for?

Reply:
I am no longer getting an extra announcement, so I changed the number back to 3.